nucypher/cli/painting.py

```diff
@@ -494,3 +499,77 @@ def paint_deployment_delay(emitter, delay: int = 3) -> None:
     for i in range(delay)[::-1]:
         emitter.echo(f"{i}...", color='yellow')
         time.sleep(1)
+
+
+def paint_stakers(emitter, stakers: List[str], agent) -> None:
+    current_period = agent.get_current_period()
+    emitter.echo(f"\nCurrent period: {current_period}")
+    emitter.echo("\n| Stakers |\n")
+    emitter.echo(f"{'Checksum address':42}  Staker information")
+    emitter.echo('=' * (42 + 2 + 53))
+
+    for staker in stakers:
+        nickname, pairs = nickname_from_seed(staker)
+        symbols = f"{pairs[0][1]}  {pairs[1][1]}"
+        emitter.echo(f"{staker}  {'Nickname:':10} {nickname} {symbols}")
+        tab = " " * len(staker)
+
+        stake = agent.owned_tokens(staker)
+        last_confirmed_period = agent.get_last_active_period(staker)
+        worker = agent.get_worker_from_staker(staker)
```

Comment:
Can this function retrieve the addresses of multiple workers associated with the same staker/stake? Re: #1302 
```
    def get_worker_from_staker(self, staker_address: str) -> str:
        worker = self.contract.functions.getWorkerFromStaker(staker_address).call()
        return to_checksum_address(worker)
```

Reply:
Our current model assumes a 1-to-1 relation between Workers and Stakers.